docs/getting-started/build-your-own-development-automation.md

```diff
@@ -1,6 +1,11 @@
-Right! You've seen some of the automation between issues, commits, builds that we provide out-of-the-box and now let's automate an action.
+Right! You've seen some of the automation between issues, commits, builds that we provide out-of-the-box and now it's time to automate some of your own development workflow and actions. We're going to:
 
-In Atomist that means writing a handler in TypeScript using the Atomist's [Rug](/user-guide/rug/index.md) support. The handler will be triggered by a certain type of activity that Atomist can detect, in this example case you want to notify the team when an issue labeled 'bug' gets fixed.
+- Develop and publish a new Atomist `handler` to listen to and take some action when issues are closed.
+- Convert an existing project of your own to be a new Atomist `generator` that can be used by the Bot to generate new projects.
```

Comment:
This is already a very long page. Perhaps it would be better to split it into two pages?